TinyConstraints/Classes/Constraints.swift

```diff
@@ -68,7 +68,7 @@ public extension Constraint {
     }
 }
 #else
-    public extension Constraint {
+@MainActor public extension Constraint {
```

Comment:
Not needed since `Constraint` is already isolated to the `@MainActor`

```suggestion
public extension Constraint {
```